src/components/form/dt-connection/dt-connection.js

```diff
@@ -183,6 +188,32 @@ export class DtConnection extends DtTags {
     return this.filteredOptions;
   }
 
+  _validateRequired() {
+    const { value } = this;
+
+    var empty = true;
+
+    if (value != undefined) {
+      console.log('value not undefined');
+      for (var i = 0; i < value.length; i++) {
+        console.log('looping');
+        if (value[i].delete == undefined) {
+          console.log('not empty');
+          empty = false;
+        }
+      }
+    }
```

Comment:
Outer if should handle undefined, null, empty.
Instead of for loop, use `value.every()`.